### DISCUSSION TITLE
Self Hosting: Is it possible to have replacation between a local minio instance and a cloud storage backend?

### DISCUSSION THREAD

**Discussion by onnyyonn** [ente-io/ente#3991] [Q&A] [answered]
As far as I understood from the docs, it's possible to enable replication between local minio instances. But I could not figure out if it's possible to mix a local minio instance and a cloud storage like B2. Is it possible at this moment? If not, will you consider supporting such configurations in the future?

**Comment by mnvr** [comment]
It is already possible. The server (museum) does not care where the S3 bucket is hosted (minio, B2 etc), so you can already mix and match.

**Reply by onnyyonn** [reply]
Thanks. I see the following in `credentials.yaml`:
```
s3:
    are_local_buckets: true
```
What should it be, when I have both minio and B2?

**Reply by mnvr** [reply, marked as answer]
false. You'll have to appropriately configure your minio so that workarounds mentioned in the [doc comment](https://github.com/ente-io/ente/blob/d25b1f7b654e8895f06a9893d98ef448d4c68e0c/server/configurations/local.yaml#L151) are not needed:
1. Run it on SSL
2. Enable `use_path_style_urls`
3. Ignore this, nothing needs to be done here since the worker will be bypassed if `worker-url` is not specified
4. Don't use `scw-eu-fr-v3` as the minio bucket so that the GLACIER storage class is not applied (you could make it any of the other two) 

Also, 2 and 4 were based on the old state of minio, maybe minio has added support for these two since then too.
